lti_tool/services/nrps.py

```diff
@@ -14,7 +14,8 @@ def fetch_member_data(context: LtiContext) -> List[dict]:
         ServiceConnector(context.deployment.registration.to_registration()),
         {"context_memberships_url": context.memberships_url},
     )
-    return nrps.get_members()
+    # TODO: Revisit type casting after pylti1p3 drops support for Python 2.
+    return cast(List[dict], nrps.get_members())
```

Comment:
Is there a more specific type for the result than just a `dict`? I'll admit that Python typing preferences are shifting much more quickly than I'm aware, but I feel like typing to specific collection classes (e.g. `dict`) was discouraged vs. using either a more generic type (e.g. `typing.Mapping[<key type>, <value type>]`) or something more restrictive (e.g. creating a `typing.TypedDict` that defined the possible keys and value types).

I'm OK leaving this as-is, but I'm curious if there was anything that suggested you go with `List[dict]` over one of the above choices.